### DISCUSSION TITLE
Improving accessibility in the Mastodon

### DISCUSSION THREAD

**Discussion by linux-lukas** [mastodon/mastodon#19230] [General]
In many ways Mastodon has become more barrier-free, but two points would certainly not be bad either: 

1 A picture description can be added to profile pictures and profile banners.
2 Posts can be read aloud (best with the help of an offline solution).

Both points would make Mastodon more usable as a platform for people with visual impairments.

**Comment by jpdevries** [comment]
👍 this initiative and also want say hello. My colleagues and I are interested in driving some pull requests to improve Mastadon accessibility.
